rotkehlchen/tests/fixtures/accounting.py

```diff
@@ -234,6 +247,7 @@ def fixture_inquirer(
         should_mock_current_price_queries=should_mock_current_price_queries,
         mocked_prices=mocked_current_prices,
         current_price_oracles_order=current_price_oracles_order,
+        ethereum_manager=None,
```

Comment:
Is this really wise? Won't this break if you try to use inquirer with a defi oracle in a test?

Reply:
Is to make things faster since this requires to setup the ethereum manager fixture and then the defi queries could be triggered and are slow. If the defi oracles are needed for some reason the new inquirer fixture with them can be used

Reply:
Then please for the love of god document this with a big comment right above this. We will forget it in a week and in a month I will change it.


```suggestion
         # EXPLAIN
        ethereum_manager=None,
```

Reply:
Still no explanation here